Common/Source/Airspace/LKAirspace.cpp

```diff
@@ -1987,8 +1987,8 @@ bool CAirspaceManager::FillAirspacesFromOpenAIP(ZZIP_FILE *fp) {
         // Airspace category
         dataStr=ASPnode.getAttribute(TEXT("CATEGORY"));
         if(dataStr==nullptr) {
-            StartupStore(TEXT(".. Skipping ASP with no CATEGORY attribute.%s"), NEWLINE);
-            continue;
+            StartupStore(TEXT("..  ASP with no CATEGORY attribute.%s"), NEWLINE);
+         //   Type = OTHER;
         }
         size_t len=_tcslen(dataStr);
```

Comment:
What Happen if dataStr == nullptr  ( cf. line 1989 ) ?

Reply:
should be type other and continue.
nota a good idea to merge last minute commits.. should we reverse?

Reply:
_tcslen(nullptr)  -> segfault